kaizen/__init__.py

```
@@ -0,0 +1,5 @@
+"""Kaizen Agent - Automated Code Improvement Tool"""
+
+from .autofix import run_autofix_and_pr
```

Ruff (0.11.9)

`.autofix.run_autofix_and_pr` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)